integration-cli/docker_cli_search_test.go

```diff
@@ -45,3 +45,49 @@ func TestSearchStarsOptionWithWrongParameter(t *testing.T) {
 
 	logDone("search - Verify search with wrong parameter.")
 }
+
+func TestSearchCmdOptions(t *testing.T) {
+	searchCmdhelp := exec.Command(dockerBinary, "search", "--help")
+	out, exitCode, err := runCommandWithOutput(searchCmdhelp)
+	if err != nil || exitCode != 0 {
+		t.Fatalf("failed to get search help information: %s, %v", out, err)
+	}
+
+	if !strings.Contains(out, "Usage: docker search [OPTIONS] TERM") {
+		t.Fatalf("failed to show docker search usage: %s, %v", out, err)
+	}
+
+	searchCmd := exec.Command(dockerBinary, "search", "busybox")
+	out_searchCmd, exitCode, err := runCommandWithOutput(searchCmd)
+	if err != nil || exitCode != 0 {
+		t.Fatalf("failed to search on the central registry: %s, %v", out_searchCmd, err)
+	}
+
+	searchCmdautomated := exec.Command(dockerBinary, "search", "--automated=true", "busybox")
+	out_searchCmdautomated, exitCode, err := runCommandWithOutput(searchCmdautomated) //The busybox is a busybox base image, not an AUTOMATED image.
+	if err != nil || exitCode != 0 {
+		t.Fatalf("failed to search with automated=true on the central registry: %s, %v", out_searchCmdautomated, err)
+	}
+
+	if strings.Contains(out_searchCmdautomated, "busybox base image") {
```

Comment:
I think "busybox" is "Busybox".
However, I think it makes more sense to search for lines that start with "busybox" (w/o a / at the end) - this way if they change the description of the default busybox this if statement will still work properly.